src/Microsoft.DotNet.Watcher.Tools/Internal/MsBuildFileSetFactory.cs

```diff
@@ -104,9 +107,16 @@ internal MsBuildFileSetFactory(ILogger logger, string projectFile, OutputSink ou
                         return fileset;
                     }
 
-                    _logger.LogError($"Error(s) finding watch items project file '{Path.GetFileName(_projectFile)}': ");
-                    _logger.LogError(capture.GetAllLines("[MSBUILD] : "));
-                    _logger.LogInformation("Fix the error to continue.");
+                    var sb = new StringBuilder()
+                        .Append("Error(s) finding watch items project file '")
+                        .Append(Path.GetFileName(_projectFile))
```

Comment:
It's a personal preference: I don't like mixing interpolation and using the string builder. But doesn't really matter either way